packages/app/app/layout.tsx

```diff
@@ -14,7 +15,10 @@ interface RootLayoutProps {
 export default function RootLayout({ children }: RootLayoutProps) {
   return (
     <html lang="en">
-      <body className="font-sans bg-surface-25">{children}</body>
+      <body className="font-sans bg-surface-25">
+        <Header />
```

Comment:
Let's call it Navbar

```suggestion
        <Navbar />
```